shared/js/background/events.es6.js

```diff
@@ -499,6 +499,44 @@ chrome.runtime.onMessage.addListener((req, sender, res) => {
 
         res(grade)
         return true
+    } else if (req.toggleSiteProtections) {
+        tabManager.setList({
+            list: 'allowlistOptIn',
+            domain: req.domain,
+            value: req.status
+        })
+        tabManager.setList({
+            list: 'allowlisted',
+            domain: req.domain,
+            value: req.status
+        })
+
+        return true
+    } else if (req.openFeedbackPage) {
+        const feedbackPath = '/html/feedback.html'
+        const url = chrome.runtime.getURL(feedbackPath)
+        chrome.tabs.create({ url: url })
+
+        return true
+    } else if (req.openBreakageReport) {
+        chrome.tabs.create({ url: req.url })
+
+        return true
+    } else if (req.breakageFlow) {
+        chrome.tabs.reload(req.tabId, { bypassCache: true }, () => {
+            setTimeout(() => {
+                chrome.tabs.executeScript(req.tabId, {
+                    code: `window.breakageModel = '${req.breakageFlow}'`
+                }, () => {
+                    console.log('execute 2')
+                    chrome.tabs.executeScript(req.tabId, {
+                        file: 'public/js/content-scripts/breakage-flow.js'
```

Comment:
If we aren't using the content-scope bundling code, I *think* we can use params here instead:
```suggestion
                        file: `public/js/content-scripts/breakage-flow.js?breakageModel=${encodeURIComponent(req.breakageFlow)}`
```

Then in the script check: `document.currentScript` params.